tuple-table/src/main/java/com/iluwatar/tuppletable/MemberDTO.java

```diff
@@ -0,0 +1,86 @@
+package com.iluwatar.tuppletable;
+
+public class MemberDTO {
```

Comment:
Use Lombok annotations to get rid of boilerplate

Reply:
Implemented, this was good feedback, I wasn't aware.